src/foam/u2/texteditor/ToolBar.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.texteditor',
+  name: 'ToolBar',
+  extends: 'foam.u2.View',
+
+  requires: [
+    'foam.u2.texteditor.DefaultTextFormats',
+    'foam.u2.texteditor.FontFace',
+    'foam.u2.texteditor.FontSize',
+    'foam.u2.texteditor.TextAlignment',
+    'foam.u2.texteditor.TextFormats',
+    'foam.u2.texteditor.ListStyle',
+    'foam.u2.texteditor.Popup',
+    'foam.u2.DetailView',
+    'foam.u2.texteditor.ToolBarItemView',
+    'foam.u2.tag.Image'
+  ],
+
+  exports: [
+    'doc_ as document'
+  ],
+
+  css: `
+    .richtext-actions {
```

Comment:
When outputting css it's typically best to use `^` to ensure we don't conflict with css classes potentially used elsewhere. `^` gets replaced with the class name to ensure the css is unique to this class. So this would get changed to something like
`^actions {`
and then when you want to use it down below, you'd replace 
`this.addClass('richtext-actions');`
with
`this.addClass(this.myClass('actions'));`